src/pages/Counter/containers/CounterContainer.js

```diff
@@ -0,0 +1,61 @@
+import { Component } from "react";
+
+import CounterView from "../components/CounterView";
+
+class CounterContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      countValue: 0,
+      isEven: true,
+    };
+  }
+
+  handleIncrement = () => {
+    this.setState((previousState) => {
+      console.log(previousState);
+      const incrementValue = previousState.countValue + 1;
+      const numericalValue =
+        incrementValue % 2 === 0 ? "Number is even" : "Number is odd";
+      return {
+        countValue: incrementValue,
+        numericalValue,
```

Comment:
what the purpose of it, you don't need to create one more field state in according with task. You must use isEven and use conditional rendering to display it